package/MDAnalysis/coordinates/chemfiles.py

```diff
@@ -161,8 +164,9 @@ def __init__(self, filename, chemfiles_format="", **kwargs):
 
         """
         if not check_chemfiles_version():
-            raise RuntimeError("Please install Chemfiles > {}"
-                               "".format(MIN_CHEMFILES_VERSION))
+            raise RuntimeError(
```

Comment:
So this one we'll only ever trigger if we go out of our way to install chemfiles < 0.9 right? can we maybe use mock/patch to spoof a non-acceptable value here?

Reply:
Test added with a mock value here: https://github.com/MDAnalysis/mdanalysis/pull/3126/files#diff-01fe53b03a7506a787f8d550d36cd7bb68074c1f1c3e3f373d76a7ab40ade459R43-R58

Reply:
Coverage is still not working for this 😕 I'm not very versed in the mixing of unittest and pytest, could this be part of the issue?

Reply:
I finally got some time to update the test to pytest, hope this fixes it!

Reply:
😕 this is still happening, I'll try to dig into what's going on a little bit later. Worst case we just merge as-is and add coverage back later.